### PR TITLE
Enable different item label for selected item

### PR DESCRIPTION
**Pull request by maximpetrov**
This pull request adds ability to display selected result (in select field) in different way than in dropdown e.g. display detail item in dropdown but only name in field after selection.

### CODE REVIEW

**Comment by gagarski** [comment]
Should fix #845

**Comment by jpic** [comment]
Looks good for me !

We just need some tests in this PR.

File: src/dal_select2/views.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ def get_results(self, context):
             {
                 'id': self.get_result_value(result),
                 'text': self.get_result_label(result),
+                'selected_text': self.get_selected_result_label(result),
```
Comment (mik-laj) [inline src/dal_select2/views.py]: I think it is worth adding this field only if it is different from ``text``. Now one text is sent in two fields even when it is identical.

Comment (jpic) [reply src/dal_select2/views.py]: That's true but we're going to merge this as is so if you got hard feelings about this then perhaps make another pr or open a ticket :joy:

**Comment by jpic** [comment]
Will be in next release.

**Pull request closed without merging**